rikai/resources/views/users/review/create.blade.php

```diff
@@ -5,57 +5,71 @@
-                                    <i class="ion-ios-star" data-index="5"></i>
-                                    <i class="ion-ios-star" data-index="6"></i>
-                                    <i class="ion-ios-star" data-index="7"></i>
-                                    <i class="ion-ios-star" data-index="8"></i>
-									<i class="ion-ios-star" data-index="9"></i>
-                                </div>
-							</div>
-						</div>
-						<div class="row">
-							<div class="col-md-2">
-								<input class="submit" type="submit" value="{{__('message.save')}}">
-							</div>
-						</div>	
-					</form>
-				</div>
-			</div>
-		</div>
-	</div>
+   <div class="container">
+      <div class="row ipad-width">
+         <div class="col-md-9 col-sm-12 col-xs-12">
+            <div class="form-style-1 user-pro" action="#">
+               <div>
+                  <h2>{{__('message.Name_Book')}}: {{$book->title}}</h2>
+               </div>
+               <br>
+               @if(count($errors)>0)
+               <div class="alert alert-danger">
+                  @foreach($errors->all() as $err)
+                  {{$err}}<br>
+                  @endforeach
+               </div>
+               @endif
+               @if (Session::has('message'))
+               <div class="flash alert-info">
+                  <p class="panel-body">
+                     {{ __(Session::get('message')) }}
+                  </p>
+               </div>
+               @endif
+               <form action="{{url('review')}}" class="user" method="post">
+                  <h4>{{__('message.Review_Book')}}</h4>
+                  <input type="hidden" name="_token" value="{{ csrf_token() }}">
+                  <input type="hidden" name="bookid" value="{{ $book->id }}">
+                  <div class="row">
+                     <div class="col-md-12 form-it">
+                        <label>{{__('message.Title')}}</label>
+                        <input type="text" placeholder="" name="title">
+                     </div>
+                  </div>
+                  <div class="row">
+                     <div class="col-md-12 form-it">
+                        <label>{{__('message.Body')}}</label>
+                        <textarea class="form-control" id="exampleFormControlTextarea1" rows="3" name="body"></textarea>
+                     </div>
+                  </div>
+                  <div class="row">
+                     <div class="col-md-6 form-it">
+                        <label>{{__('message.Rated_books')}} </label>
+                        <div class="container1">
+                           <div class="star-widget">
+                              <?php
+                                 for($i=1;$i<11;$i++){
+                                 $input = rateform($i);
+                                 echo $input;
+                                 }
+                              ?>
```

Comment:
Vòng for này xử lý trực tiếp trong helper luôn.
Ngoài này chỉ việc call function rateform() ra thôi